packages/artemisrgb-bin/artemisrgb-bin.pacscript

```diff
@@ -0,0 +1,41 @@
+pkgname=artemisrgb-bin
+pkgver=1.2025.0819.0
+gives="artemisrgb"
+url="https://artemis-rgb.com/"
+pkgdesc="A universal RGB control software"
+arch=("x86_64")
+provides=("artemisrgb")
+conflicts=("artemisrgb" "artemisrgb-git")
+license=("custom:PolyForm Noncommercial License 1.0.0")
+depends=("dotnet-runtime-8.0" "hicolor-icon-theme" "libc6" "bash" "zlib1g" "fontconfig" "libstdc++6")
```

Comment:
Looking at the CI output, would a lower version of dotnet work or must it be 8.0?

Reply:
You should check.